### PR TITLE
[IMP] purchase_order_weight_volume: added license

### PR DESCRIPTION
**Pull request by ilyasProgrammer**

### CODE REVIEW

File: purchase_order_weight_volume/__manifest__.py [hunk 1]
```diff
@@ -1,10 +1,13 @@
+# © 2023 ooops404
```
Comment (simahawk) [inline purchase_order_weight_volume/__manifest__.py]: pls replace the copyright symbol w/ "Copyright"

File: purchase_order_weight_volume/__manifest__.py [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
+# License AGPL-3 - See https://www.gnu.org/licenses/agpl-3.0.html
 {
     "name": "Purchase Order Weight and Volume",
     "summary": """Display purchase order weight and volume""",
     "version": "14.0.1.0.0",
     "author": "Ilyas, Ooops404, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/purchase-workflow",
     "category": "Purchase Management",
+    "maintainers": ["ilyasProgrammer"],
```
Comment (simahawk) [inline purchase_order_weight_volume/__manifest__.py:10]: you cannot merge yet, this must be in the destination branch already :wink:

**Review by simahawk — CHANGES_REQUESTED** [review]

File: purchase_order_weight_volume/__manifest__.py (hunk 1 above)
Comment (ilyasProgrammer) [reply purchase_order_weight_volume/__manifest__.py]: ok

File: purchase_order_weight_volume/__manifest__.py (hunk 2 above)
Comment (ilyasProgrammer) [reply purchase_order_weight_volume/__manifest__.py:10]: got it )

**Comment by simahawk** [comment]
/ocabot merge patch

Can you fwd port to 15 pls?

**Review by simahawk — APPROVED** [review]

**Pull request merged**